### PR TITLE
vidcutter: init at 6.0.5.3

### PR DESCRIPTION
**Pull request by zi3m5f**
Closes package request: #353351

It builds and the gui starts but not tested further.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by mausch** [comment]
I'm not really familiar with vidcutter, I just wanted to do some casual editing (I use this sort of tool maybe once a year?).
I ended up using kdenlive but I gave this a try and it seems to be working fine up to bugs in the app itself.
For example I hit this bug https://github.com/ozmartian/vidcutter/issues/253#issuecomment-553965906
I think this is worth merging :+1:

File: pkgs/by-name/vi/vidcutter/package.nix [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  buildInputs = [
+    mpv
+    qt5.qtwayland
+  ];
+
+  propagatedBuildInputs = [
+    ffmpeg
+    mediainfo
+  ];
+
```
Comment (Aleksanaa) [inline pkgs/by-name/vi/vidcutter/package.nix:37]:
```suggestion
```

File: pkgs/by-name/vi/vidcutter/package.nix [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  dependencies = with python3Packages; [
+    pyopengl
+    pyqt5
+    simplejson
+  ];
+
+  dontWrapQtApps = true;
+  makeWrapperArgs = [ "\${qtWrapperArgs[@]}" ];
```
Comment (Aleksanaa) [inline pkgs/by-name/vi/vidcutter/package.nix:49]:
```suggestion
  makeWrapperArgs = [
    "\${qtWrapperArgs[@]}"
    "--prefix PATH : ${lib.makeBinPath [ ffmpeg mediainfo ]}"
  ];
```

**Review by Aleksanaa — APPROVED** [review]
Just recommended:

File: pkgs/by-name/vi/vidcutter/package.nix [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+  meta = {
+    description = "Modern yet simple multi-platform video cutter and joiner";
+    longDescription = ''
+      A modern, simple to use, constantly evolving and hella fast MEDIA CUTTER + JOINER
+      with frame-accurate SmartCut technology, chapter support, media stream selection for audio + subtitle channels
+      and blackdetect video filter support to automatically detect scene changes or skip commercials in digital TV recordings.
+    '';
+    homepage = "https://vidcutter.ozmartians.com/";
+    changelog = "https://github.com/ozmartian/vidcutter/blob/master/CHANGELOG";
```
Comment (pluiedev) [inline pkgs/by-name/vi/vidcutter/package.nix:62]: Changelog should point to current version
```suggestion
    changelog = "https://github.com/ozmartian/vidcutter/blob/refs/tags/${version}/CHANGELOG";
```

**Review by pluiedev — APPROVED** [review]
LGTM except for one thing

File: pkgs/by-name/vi/vidcutter/package.nix (hunk 3 above)
Comment (pluiedev) [reply pkgs/by-name/vi/vidcutter/package.nix:62]: I *guess* there's no consensus but then IMO the point of a changelog is to see what's different in the current version compared to older ones. If it's pinned to master then you might see future versions that are not yet packaged (including alpha/beta/RC builds), which is not exactly super helpful.

Comment (Aleksanaa) [reply pkgs/by-name/vi/vidcutter/package.nix:62]: Not blocking as the consensus isn't being made. Another view emphasizes the completeness and clickability of the changelog and the introduction of version to add additional complexity.

**Pull request merged**